docs/sources/clients/promtail/troubleshooting.md

```diff
@@ -19,6 +19,31 @@ To start Promtail in dry run mode use the flag `--dry-run` as shown in the examp
 cat my.log | promtail --stdin --dry-run --client.url http://127.0.0.1:3100/loki/api/v1/push
 ```
 
+## Inspecting
+
+Promtail can be configured to print all changes to log entries as each pipeline stage is executed (via the flag `--inspect`).
+Each log entry contains four fields:
+- line
+- timestamp
+- labels
+- extracted fields
+
+Whenever any of these change, a log message will be printed:
```

Comment:
```suggestion
Enable the inspection output using the `--inspect` command-line option. This option can be used in combination with `--stdin` and `--dry-run`.
```

Reply:
I think we need to give the relevant flag here (`--inspect`) more visual hierarchy so it can be found more easily. Any objection to my adding a linebreak between those 2 sentences you suggested?

Reply:
Yes.  If you were to put a line break inbetween, then the second sentence must change.  It starts with "This option", and so loses the reference to what would then be the previous paragraph.  If you want to add the newline, change the second sentence (new paragraph) to start with 
> The `--inspect` option can be . . .